recipes/libs/zstd.yaml

```
@@ -0,0 +1,35 @@
+inherit: [cpackage, make, install, patch, strip]
+
+metaEnvironment:
+  PKG_VERSION: "1.5.6"
+
+checkoutSCM:
+  scm: url
+  url: ${GITHUB_MIRROR}/facebook/zstd/releases/download/v${PKG_VERSION}/zstd-${PKG_VERSION}.tar.gz
+  stripComponents: 1
+
+checkoutScript: |
```

A checkout script must be marked explicitly as deterministic. Otherwise Bob will always re-run the checkout script.

```suggestion
checkoutDeterministic: True
checkoutScript: |
```